### PR TITLE
Make kubemark work on Jenkins

### PR DESCRIPTION
**Pull request by gmarek**

### CODE REVIEW

File: test/kubemark/start-kubemark.sh [hunk 1]
```diff
@@ -23,9 +24,26 @@ source "${KUBE_ROOT}/cluster/kubemark/util.sh"
...
+MAKE_DIR="${KUBE_ROOT}/cluster/images/kubemark"
+
+if [ "${RUN_FROM_DISTRO}" == "false" ]; then
+  # Running from repository
+  cp "${KUBE_ROOT}/_output/release-stage/server/linux-amd64/kubernetes/server/bin/kubemark" "${MAKE_DIR}"
+else
+  cp "${KUBE_ROOT}/server/kubernetes-server-linux-amd64.tar.gz" "."
+  tar -xzf kubernetes-server-linux-amd64.tar.gz
+  cp "kubernetes/server/bin/kubemark" "${MAKE_DIR}"
+  ls
```
Comment (wojtek-t) [inline test/kubemark/start-kubemark.sh]: seems unneded

File: test/kubemark/start-kubemark.sh [hunk 2]
```diff
@@ -23,9 +24,26 @@ source "${KUBE_ROOT}/cluster/kubemark/util.sh"
...
+  ls ${MAKE_DIR}
```
Comment (wojtek-t) [inline test/kubemark/start-kubemark.sh]: is this needed?

File: test/kubemark/start-kubemark.sh [hunk 3]
```diff
@@ -49,17 +67,26 @@ gcloud compute instances create ${MASTER_NAME} \
...
+    "${KUBE_ROOT}/_output/release-tars/kubernetes-server-linux-amd64.tar.gz" \
+    "${KUBE_ROOT}/test/kubemark/start-kubemark-master.sh" \
+    "${KUBE_ROOT}/test/kubemark/configure-kubectl.sh" \
+    "hollow-cluster-master":~
+else
+  gcloud compute copy-files --zone="${ZONE}" \
+    "${KUBE_ROOT}/server//kubernetes-server-linux-amd64.tar.gz" \
+    "${KUBE_ROOT}/test/kubemark/start-kubemark-master.sh" \
+    "${KUBE_ROOT}/test/kubemark/configure-kubectl.sh" \
+    "hollow-cluster-master":~
```
Comment (wojtek-t) [inline test/kubemark/start-kubemark.sh]: The last three files seems the same in both cases - can you copy them unconditionally?

File: test/kubemark/start-kubemark.sh [hunk 4]
```diff
@@ -49,17 +67,26 @@ gcloud compute instances create ${MASTER_NAME} \
...
-  hollow-cluster-master:~
+if [ "${RUN_FROM_DISTRO}" == "false" ]; then
+  gcloud compute copy-files --zone="${ZONE}" \
+    "${KUBE_ROOT}/_output/release-tars/kubernetes-server-linux-amd64.tar.gz" \
+    "${KUBE_ROOT}/test/kubemark/start-kubemark-master.sh" \
+    "${KUBE_ROOT}/test/kubemark/configure-kubectl.sh" \
+    "hollow-cluster-master":~
+else
+  gcloud compute copy-files --zone="${ZONE}" \
+    "${KUBE_ROOT}/server//kubernetes-server-linux-amd64.tar.gz" \
```
Comment (wojtek-t) [inline test/kubemark/start-kubemark.sh]: Is "//" expected? Should it be "/" instead?

File: test/kubemark/start-kubemark.sh (hunk 1 above)
Comment (gmarek) [inline test/kubemark/start-kubemark.sh]: Sorry, debug.

File: test/kubemark/start-kubemark.sh (hunk 2 above)
Comment (gmarek) [inline test/kubemark/start-kubemark.sh]: Nope, it's a debug.

File: test/kubemark/start-kubemark.sh (hunk 3 above)
Comment (gmarek) [inline test/kubemark/start-kubemark.sh]: I can, but actually establishing connection is longer than copying small files when the connection is already established. It's faster this way, even if it's uglier.

File: test/kubemark/start-kubemark.sh (hunk 4 above)
Comment (gmarek) [inline test/kubemark/start-kubemark.sh]: Yup. Thanks - I wonder how it's possible that it worked;)

**Comment by wojtek-t** [comment]
LGTM

**Pull request merged**